tests/e2e.py

```diff
@@ -0,0 +1,117 @@
+import unittest
+import datetime
+
+import pandas.testing
+
+from dune_client.models import ResultsResponse
+from sqlalchemy import create_engine
+from pandas import DataFrame
+
+from src.main import save_to_postgres, dune_result_to_df
+from tests.db_util import query_pg
+from sqlalchemy import BIGINT, BOOLEAN, VARCHAR, DATE, TIMESTAMP
+from sqlalchemy.dialects.postgresql import BYTEA
+
+DB_URL = "postgresql://postgres:postgres@localhost:5432/postgres"
```

Comment:
Would be convenient to accept env vars for the DB user, password, name components, with the present values as defaults.
```suggestion
from os import getenv

DB_URL = f"postgresql://{getenv('DB_USER', 'postgres')}:{getenv('DB_PASSWORD', 'postgres')}@localhost:5432/{getenv('DB_NAME', 'postgres')}""
```

Reply:
Looks but the line is too wide. I'd also prefer the db url to be one env var instead of a bunch of different things.

Reply:
Sure, having an env var getter for the whole connection string would be cool too.

Reply:
Done.